index.js

```diff
@@ -0,0 +1,108 @@
+// import Book from modules
+import addToLocalStorage from './modules/localStorage.js';
+import Book from './modules/shelfClass.js';
+
+let books;
+
+const book1 = new Book('title1', 'author1', '001');
+const book2 = new Book('title2', 'author2', '002');
+const book3 = new Book('title3', 'author3', '003');
+
+books = [book1, book2, book3];
+
+// add event listner to newly added book remove button
+let removeBtn = document.querySelectorAll('.remove');
+
+function removeBook(ref) {
+  const result = books.filter((value) => value.id !== ref);
+  books = result;
+  addToLocalStorage(books);
+}
+
+function addEvents() {
+  removeBtn.forEach((element) => {
+    element.addEventListener('click', (e) => {
+      const ref = e.target.id;
+      removeBook(ref);
+      e.target.parentElement.remove();
+    });
+  });
+}
+
+// populate dom with the list
+const addBtn = document.querySelector('#add-btn');
+function showBooks(list) {
+  const booksListDiv = document.querySelector('#Book_Lists');
+  for (let i = 0; i < list.length; i += 1) {
+    const div = document.createElement('div');
+    const pTitle = document.createElement('p');
+    pTitle.innerHTML = `"${list[i].title}" by ${list[i].author}`;
+    div.appendChild(pTitle);
+    const btn = document.createElement('button');
+    btn.className = 'remove';
+    btn.innerHTML = 'Remove';
+    btn.id = list[i].id;
+    div.appendChild(btn);
+    div.className = 'book_lists';
+    booksListDiv.appendChild(div);
+  }
+  removeBtn = document.querySelectorAll('.remove');
+  addEvents();
+}
+const listClass = document.getElementById('List');
+const addNewClass = document.getElementById('Add_new');
+const contactClass = document.getElementById('Contact');
+
+// add book to the list
+function addBook() {
```

Comment:
- [ ] Please, consider using arrow function syntax, as required in the project (use ES6 syntax). For example, here you can go on with 
```
const addBook = () => {
....
```

- [ ] Also, will be better to create an additional module for adding the books method, then import it here in the `index.js` file